### PR TITLE
Changed OpSec bugzilla product/component to Enterprise Information Se…

### PR DESCRIPTION
**Pull request by gdestuynder**
New opsec product/component

### CODE REVIEW

**Comment by gdestuynder** [comment]
Note: also simplified URLs for edits

File: Security/OpSec.mediawiki [hunk 1]
```diff
@@ -81,8 +81,7 @@ The goal of the NSM project is to provide useful context information for both th
 : business hours availability. 1 year data retention.
 ; Costs
 : platform supported by OpSec. Subscriber’s handles the cost of provisioning and monitoring the agents on target systems.
-; Service request
-: [https://bugzilla.mozilla.org/enter_bug.cgi?assigned_to=jvehent%40mozilla.com&blocked=896480&bug_file_loc=http%3A%2F%2F&bug_ignored=0&bug_severity=normal&bug_status=NEW&cf_blocking_b2g=---&cf_fx_iteration=---&cf_fx_points=---&component=Operations%20Security%20(OpSec)%3A%20MIG&contenttypemethod=autodetect&contenttypeselection=text%2Fplain&defined_groups=1&flag_type-4=X&flag_type-607=X&flag_type-800=X&flag_type-803=X&form_name=enter_bug&groups=infra&maketemplate=Remember%20values%20as%20bookmarkable%20template&op_sys=Linux&priority=--&product=mozilla.org&rep_platform=x86_64&short_desc=MIG%20Service%20request%20for%20%3Cteam%20name%3E&target_milestone=---&version=other request bug]
+: [https://bugzilla.mozilla.org/enter_bug.cgi?product=Enterprise%20Information%20Security&component=MIGjvehent%40mozilla.com&blocked=896480 request bug]
```
Comment (ameihm0912) [inline Security/OpSec.mediawiki]: Should this include assigned_to?

File: Security/OpSec.mediawiki [hunk 2]
```diff
@@ -104,7 +103,7 @@ OpSec operates a client/server platform to facilitate the investigation of large
 ; Costs
 : platform supported by OpSec. Subscriber’s handles the cost of provisioning and monitoring the agents on target systems.
 ; Service request
-: [https://bugzilla.mozilla.org/enter_bug.cgi?assigned_to=jvehent%40mozilla.com&blocked=896480&bug_file_loc=http%3A%2F%2F&bug_ignored=0&bug_severity=normal&bug_status=NEW&cf_blocking_b2g=---&cf_fx_iteration=---&cf_fx_points=---&component=Operations%20Security%20(OpSec)%3A%20General&contenttypemethod=autodetect&contenttypeselection=text%2Fplain&defined_groups=1&flag_type-4=X&flag_type-607=X&flag_type-800=X&flag_type-803=X&form_name=enter_bug&groups=infra&maketemplate=Remember%20values%20as%20bookmarkable%20template&op_sys=Linux&priority=--&product=mozilla.org&rep_platform=x86_64&short_desc=TDS%20Service%20request%20for%20%3Cteam%20name%3E&target_milestone=---&version=other request bug]
+: [https://bugzilla.mozilla.org/enter_bug.cgi?product=Enterprise%20Information%20Security&component=MIGjvehent%40mozilla.com&blocked=896480 request bug]
```
Comment (ameihm0912) [inline Security/OpSec.mediawiki]: Same here

**Comment by gdestuynder** [comment]
r+ from alm (IRC)

**Pull request merged**